Installing Arkime stable versions from source produces error due to git

**Describe the bug**
If a user wants to build and install a **stable** version from source, `make install` will use `git` command that cannot be effective since stable versions are not git repositories.

Specifically, download a stable version, for example the latest one:
```
wget https://github.com/arkime/arkime/archive/v5.6.0.tar.gz
tar -zxvf v5.6.0.tar.gz
cd arkime-5.6.0
```
Then, after building it, when you run:
```
make install
```
it produces the following error:
```

fatal: not a git repository (or any parent up to mount point /)
Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).
node:child_process:966
    throw err;
    ^

Error: Command failed: git describe --tags
fatal: not a git repository (or any parent up to mount point /)
Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).

    at genericNodeError (node:internal/errors:984:15)
    at wrappedFn (node:internal/errors:538:14)
    at checkExecSyncError (node:child_process:891:11)
    at Object.execSync (node:child_process:963:15)
    at git (/home/athena/Downloads/arkime-5.6.0/common/git.js:14:24)
    at Object.<anonymous> (/home/athena/Downloads/arkime-5.6.0/viewer/vueapp/build/webpack.prod.conf.js:44:37)
    at Module._compile (node:internal/modules/cjs/loader:1469:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1548:10)
    at Module.load (node:internal/modules/cjs/loader:1288:32)
    at Module._load (node:internal/modules/cjs/loader:1104:12) {
  status: 128,
  signal: null,
  output: [
    null,
    '',
    'fatal: not a git repository (or any parent up to mount point /)\n' +
      'Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).\n'
  ],
  pid: 166931,
  stdout: '',
  stderr: 'fatal: not a git repository (or any parent up to mount point /)\n' +
    'Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).\n'
}

Node.js v20.18.1
make[1]: *** [Makefile:27: install] Error 1
make[1]: Leaving directory '/home/athena/Downloads/arkime-5.6.0/viewer'
make: *** [Makefile:456: install-recursive] Error 1
```
Of course it will work only if Arkime is cloned directly by `git clone`, but if a user wants to use a stable version, for example downloaded by the Release section, the error is produced because the stable archives are not git repositories.

A solution should be to not use git command during the install. Probably the usage of submodules could solve?


**Elasticsearch version**: 7.10.2

**Arkime/Moloch version**: All stable versions

**OS name and version**: Arch Linux rolling

--- a/easybutton-build.sh
+++ b/easybutton-build.sh
@@ -162,6 +162,9 @@ command -v sudo >/dev/null 2>&1 || { echo >&2 "ARKIME: sudo is required to be in
 # Check if in right directory
 [ -f "./easybutton-build.sh" ] || { echo >&2 "ARKIME: must run from arkime directory"; exit 1; }
 
+# Check if git
+[[ -d .git || "$ARKIME_BUILD_FULL_VERSION" != "" ]] || { echo >&2 "ARKIME: must run from 'git clone' directory or use something like 'export ARKIME_BUILD_FULL_VERSION=vVERSION_STRING'"; exit 1; }
+
 MAKE=make
 UNAME="$(uname)"
 

